app/controllers/api/movies_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::MoviesController < ApplicationController
+  require 'rest-client'
+
+  def index
+    netflix = Rails.application.credentials.unogsng_key
+    response = RestClient.get('https://unogsng.p.rapidapi.com/search?type=movie&orderby=rating',
+                              headers = { 'x-rapidapi-key': netflix })
+    results = JSON.parse(response)['results'].select{ |film| film["avgrating"] > 4 }
```

Comment:
Since it is the "top 10" i think you should also sort the movies after selecting all with higher rating than 4. 
So something like: `results.sort_by{ |film| film['avgrating']}` or something like it so you sort the array in either an ascending order or even more preferably a descending order, and for that you just call `reverse` on the example above

Reply:
We had movies sorted by IMDB rating, but sorting them by NETFLIX rating makes more sense, we will do it